pkg/kubernetes/pkg/util/metrics/metrics.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"net/http"
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var registerMetrics sync.Once
+
+// Metrics are identified in Prometheus by concatinating Namespace, Subsystem and Name
+// omitting any nulls and separating each with an underscore
+const (
+	CatalogNamespace = "servicecatalog"
+)
+
+var (
+	BrokerServiceClassCount = prometheus.NewGaugeVec(
```

Comment:
Since we are exposing this variable publicly, need comments describing what they are.